src/m365/purview/commands/retentionlabel/retentionlabel-add.ts

```diff
@@ -179,6 +208,25 @@ class PurviewRetentionLabelAddCommand extends GraphCommand {
       this.handleRejectedODataPromise(err);
     }
   }
+
+  private async getEventType(args: CommandArgs): Promise<string> {
+    if (args.options.eventTypeId) {
+      return args.options.eventTypeId;
+    }
+
```

Comment:
Let's add a verbose logging line saying that we're retrieeving the event type id or something like that.

Reply:
Done 😄 
![image](https://user-images.githubusercontent.com/35696168/224385359-4306fdd4-fcc8-4448-aed4-d58c2e8a6936.png)